django/contrib/admin/options.py

```diff
@@ -436,7 +436,7 @@ def get_sortable_by(self, request):
             else self.get_list_display(request)
         )
 
-    def lookup_allowed(self, lookup, value):
+    def lookup_allowed(self, lookup, value, request):
```

Comment:
Folks can call `super().lookup_allowed(...)` in subclasses, so for now I'd use `def lookup_allowed(self, lookup, value, request=None):` signature:
```suggestion
    # RemovedInDjango60Warning: when the deprecation ends, replace with:
    # def lookup_allowed(self, lookup, value, request):
    def lookup_allowed(self, lookup, value, request=None):
```